recipes/r-evalhyd/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set version = "0.1.0.0" %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-evalhyd
+  version: {{ version }}
+
+source:
+  url: https://gitlab.irstea.fr/HYCAR-Hydro/evalhyd/evalhyd-r/-/archive/v{{ version }}/evalhyd-r-v{{ version }}.tar.gz
+  sha256: dc4eda2655182bf9c7189e969f398aca053c46128997c8fd2fe730691e1eaa65
+
+build:
+  number: 0
+  merge_build_host: True          # [win]
+  script_env:
+    - EVALHYD_R_VENDOR_XTL=FALSE
+    - EVALHYD_R_VENDOR_XTENSOR=FALSE
+    - EVALHYD_R_VENDOR_XTENSOR_R=FALSE
+    - EVALHYD_R_VENDOR_EVALHYD_CPP=FALSE
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'                   # [win]
```

Comment:
This needs a comment IMO; what's the reason we need this exception?

Reply:
Fair point. I have added a comment, referring to a PR on conda-build, explanation why the missing R.dll message needs to be ignored.